docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -4,5 +4,12 @@
 The changes listed below have been released for the first time in {es}
 8.0.0-alpha1.
 
+[[breaking-8.0.0-alpha1]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* disallow specifying the same percentile multiple times in percentiles aggregation {pull}52257[#52257]
+
```

Comment:
@lcawl I added this release note following how its done in 7.x, does it look ok to you?